fix(snowflake): Remove invalid operations from registry (#5182)
ibis-project/ibis: docs/backends/support_matrix.md

Backends are shown in descending order of the number of supported operations.

> !!! tip "The backend with low coverage is a good place to start contributing!"

Reviewer: ```suggestion
!!! tip "Backends with low coverage are good places to start contributing!"
```

    Each backend implements operations differently, but this is usually very similar

>     to other backend.

Reviewer: ```suggestion
    to other backends.
```

>     If you want to start contributing to this project, it's a good idea to start by adding missing operations

Reviewer: ```suggestion
    If you want to start contributing to ibis, it's a good idea to start by adding missing operations
```
